.github/workflows/release.yaml

```diff
@@ -183,6 +183,9 @@ jobs:
   build-ios:
     runs-on: macos-latest
     steps:
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest-stable
```

Comment:
Copy-pasting code from commits is a bad practice and may lead to merge conflicts. You could have considered merging or cherry-picking commits.

Reply:
Sorry...
I'll make a note for next time